docs/COMPATIBILITY_MATRIX.md

```diff
@@ -0,0 +1,39 @@
+| v1.8.2                              |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |                    |                    |           |           |
+| v1.9.0                              |                    |                    |                    |     :warning:      |     :warning:      |     :warning:      |     :warning:      |     :warning:      |           |           |
+| v1.9.1                              |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |     :warning:      |           |           |
+| v1.10.0                             |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |           |           |
+| v1.11.0                             |                    |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: |           |
+| v1.11.1                             |                    |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: |           |
+| v1.11.2                             |                    |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: |           |
+| v1.12.0                             |                    |                    |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :warning: | :warning: |
+
+- :white_check_mark: Compatible
+- :warning: Has issues
+- :x: Incompatible
+
+## Warning
+
+- :warning: : module version: `v1.9.0` and Kubernetes Version: `1.20.x`. It works as expected. Marked as a warning
+because it is not officially supported by [SIGHUP](https://sighup.io).
+- :warning: : module version: `v1.9.0`. Has some problems exposing metrics and with the log recollection.
+Move to v1.9.1 instead. [Follow the migration path.](docs/releases/v1.9.1.md)
+- :warning: : module version: `v1.10.0` and Kubernetes Version: `1.21.x`. It works as expected. Marked as a warning
+because it is not officially supported by [SIGHUP](https://sighup.io).
+- :warning: : module version: `v1.11.X` and Kubernetes Version: `1.22.x`. This version is not fully supported since the component
+  `forecastle` is not supported in k8s version `1.22`
+- :warning: : module version: `v1.12.0` and Kubernetes Version: `1.23.x`. This version is not fully supported since the component
```

Comment:
We can ideally remove this now. Forecastle now works! sorry I forgot to remove this